tests/scripts/generate.feature

```diff
@@ -3,6 +3,7 @@ Feature: Code generation
     Scenario: Given and when using the same fixture should not evaluate it twice
         Given I have an empty list
         And 1 have a fixture (appends 1 to a list) in reuse syntax
+        And There's a step with a quote
```

Comment:
This should be a test by itself, let's not hook in other tests.

Let's also test that it behaves correctly in case of double quotes.

Reply:
Ok, where do I put it? As another scenario in generate.feature, or a new feature to check the quote escaping of generate (`generate_escape_quotes.feature` or something)?

Reply:
Added a separate generate_with_quotes.feature and added a testcase in test_generate.py